data/operators/man_made/pipeline.json

```
@@ -1966,11 +1957,13 @@
     },
     {
       "displayName": "Exolum",
-      "id": "exolum-143312",
-      "locationSet": {"include": ["001"]},
+      "id": "exolum-0a3816",
+      "locationSet": {"include": ["es", "gb"]},
+      "matchNames": ["clh", "gpss"],
```

```suggestion
      "matchNames": ["clh", "clh-ps", "gpss"],
```
Do you need to add `CLH-PS` also as a matchName?

---

Probably not necessary (it's technically the name of the pipeline system not the operator) but it is present a few times in the UK so I've added it.